pycoast/cw_agg.py

```diff
@@ -61,23 +61,35 @@ def _draw_polygon(self, draw, coordinates, **kwargs):
 
     def _draw_rectangle(self, draw, coordinates, **kwargs):
         """Draw rectangle."""
-        pen = aggdraw.Pen(kwargs['outline'])
+        pen = aggdraw.Pen(kwargs['outline'],
+                          kwargs['width'],
+                          kwargs['outline_opacity'])
 
-        fill_opacity = kwargs.get('fill_opacity', 255)
+        if kwargs['fill'] is None:
+            fill_opacity = 0
+        else:
+            fill_opacity = kwargs['fill_opacity']
```

Comment:
Why the change in default? And why get rid of `.get`?

Reply:
I think it is something I played with but forgot to restore it back. Will modify it in next push.